docs/parameters.md

```diff
@@ -389,7 +389,7 @@ lead to a sensitivity failure, and instead propose a solution whose objective-fu
 network is converging for all contingency scenarios.
 
 #### sensitivity-parameters
-- **Expected value**: SensitivityComputationParameters ([PowSyBl](https://www.powsybl.org/pages/documentation/simulation/sensitivity/) configuration)
+- **Expected value**: SensitivityComputationParameters ([PowSyBl](inv:powsyblcore:std:doc#simulation/sensitivity/configuration) configuration)
```

Comment:
Here also I pointed to the configuration documentation of sensitivity, and not just the sensitivity documentation.